config/routes.php

```diff
@@ -53,19 +53,41 @@
     // API group.
     // By default it responds with XML regardless of content-type.
     // Individual sub-routes are responding with JSON.
-    Group::create('/api', [
-        Route::get('/info/v1', function (DataResponseFactoryInterface $responseFactory) {
-            return $responseFactory->createResponse(['version' => '1.0', 'author' => 'yiisoft']);
-        })->name('api/info/v1'),
-        Route::get('/info/v2', ApiInfo::class)
-            ->addMiddleware(FormatDataResponseAsJson::class)
-            ->name('api/info/v2'),
-        Route::get('/user', [ApiUserController::class, 'index'])
-            ->name('api/user/index'),
-        Route::get('/user/{login}', [ApiUserController::class, 'profile'])
-            ->addMiddleware(FormatDataResponseAsJson::class)
-            ->name('api/user/profile'),
-    ])->addMiddleware(ApiDataWrapper::class)->addMiddleware(FormatDataResponseAsXml::class),
+//    Group::create('/api', [
```

Comment:
We need to remove it, not just comment.

Reply:
> We need to remove it, not just comment.

I'll do it when override all routes with a new syntax.